tests/Unit/Domain/CoordinateMaps/Test_CubicScale.cpp

```diff
@@ -0,0 +1,265 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <array>
+#include <cstddef>
+#include <string>
+#include <unordered_map>
+
+#include "ControlSystem/FunctionOfTime.hpp"
+#include "ControlSystem/PiecewisePolynomial.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/CoordinateMaps/CubicScale.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/StdArrayHelpers.hpp"
+#include "Utilities/TypeTraits.hpp"
+#include "tests/Unit/Domain/CoordinateMaps/TestMapHelpers.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+SPECTRE_TEST_CASE("Unit.Domain.CoordMapsTimeDependent.CubicScale",
+                  "[Domain][Unit]") {
+  constexpr size_t deriv_order = 2;
+
+  const auto& run_tests = [=](
```

Comment:
`[deriv_order](....`

Reply:
no `&`

Reply:
In regards to `[=]` as opposed to `[deriv_order]` was due to gcc not compiling in the second case, despite clang being happy (as usual). My new proposed solution is to make `deriv_order` a 
`static constexpr`, in which case it is implicitly captured and both compilers are happy. Let me know if there is some reason this is a bad choice.

Reply:
Not that I can think of, especially in a test :) Thanks for letting us know!